netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -251,21 +251,27 @@ static GENERIC_MAPPING _net_ebpf_ext_generic_mapping = {0};
 //
 // sock_addr helper functions.
 //
-static uint32_t
-_get_process_id()
-{
-    return (uint32_t)(uintptr_t)PsGetCurrentProcessId();
-}
-
 static uint32_t
 _get_thread_id()
 {
     return (uint32_t)(uintptr_t)PsGetCurrentThreadId();
 }
 
 static uint64_t
-_ebpf_sock_addr_get_current_pid_tgid(_In_ const bpf_sock_addr_t* ctx)
+_ebpf_sock_addr_get_current_pid_tgid(
```

Comment:
question: why did we have to change this? this will cause any older programs compiled with eBPF version <= 0.17.1 to fail to load if they are using this helper function.

Reply:
restored.